db/seeds.rb

```diff
@@ -11,7 +11,7 @@
 comedy = Category.create(title: 'Comedy')
 action = Category.create(title: 'Action')
 
-Video.create(title: 'Inception', 
+inception = Video.create(title: 'Inception', 
```

Comment:
When you do `rake db:seed` and you have invalid records to be created with `create` it will not notify you. Exception will tell you what happened. Thats why it is far superior here.